src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -530,6 +619,86 @@ public void testCanAnnotateMixedContigHg19Clinvar() {
                 .count(), NUM_CLINVAR_HITS, "Found unexpected number of ClinVar hits!");
     }
 
+    @Test
+    public void testManualAnnotationsCorrectness() {
```

Comment:
Add comment describing what this is testing.

Reply:
Fixed.